### PR TITLE
Always show the support link

### PR DESCRIPTION
**Pull request by huwshimi**
Fixes: #3073. Fixes: #3072. Fixes: #3078.

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- in gisf and gij click the help icon in the header
- the "Get support" link should be visible
- hover the applications/machines buttons
- the cursor should be the hand pointer thingy

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7882/

File: jujugui/static/gui/src/app/components/header-help/header-help.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ class HeaderHelp extends React.Component {
   _generateIssuesLink() {
     let label = 'File Issue';
     let link = 'https://github.com/juju/juju-gui/issues';
-    if (this.props.gisf && this.props.user) {
+    if (this.props.user) {
```
Comment (frankban) [inline jujugui/static/gui/src/app/components/header-help/header-help.js]: I am not sure about pointing gijoe users to the JAAS support page. I'd like them not to assume that we provide support for their own controllers.

Comment (hatched) [inline jujugui/static/gui/src/app/components/header-help/header-help.js]: @frankban what if we created another landing page that was identical but with some of the wordings/links changed? But in the mean time updating the link to point to https://jujucharms.com/docs/stable/about-juju in the gisf case?

Comment (frankban) [inline jujugui/static/gui/src/app/components/header-help/header-help.js]: Yes, let's add a link to that page in the gijoe case.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7906/

File: jujugui/static/gui/src/app/components/header-help/header-help.js (hunk 1 above)
Comment (huwshimi) [inline jujugui/static/gui/src/app/components/header-help/header-help.js]: I've added the alternate link for gijoe.

File: jujugui/static/gui/src/app/components/header-help/header-help.js [hunk 2]
```diff
@@ -58,9 +58,10 @@ class HeaderHelp extends React.Component {
...
       label = 'Get Support';
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/header-help/header-help.js]: I'm not sure that this label makes sense in the gijoe case. Maybe "Documentation" is better in that case?

**Comment by hatched** [comment]
Thanks @huwshimi 

Everything works great, just one small request about the label then this is good to land!

**Comment by huwshimi** [comment]
@hatched there is a separate "View documentation" link, so instead of having a different gijoe link for support it should be a different link for documentation?

**Comment by hatched** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**